soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -338,138 +338,138 @@ void InitTrickNames() {
      Text{"Progressive Stamina Meter", "Jauge d'endurance (prog.)", "Medidor de vigor progresivo"},
-     Text{"Progressive Energy Meter", "Jauge d'énergie (prog.)", "Medidor de energía progresivo"},
-     Text{"Progressive Magic Powder", "Poudre magique (prog.)", "Medidor de carga progresivo"}};
+     Text{"Progressive Energy Gauge", "Jauge d'énergie (prog.)", "Medidor de energía progresivo"},
+     Text{"Progressive Stamina Wheel", "", ""}};
   trickNameTable[0x8B] = {
-     Text{"Progressive Memento", "Souvenir (prog.)", "Silbato progresivo"},
+     Text{"Progressive Whistle", "Siffler (prog.)", "Silbido progresivo"},
      Text{"Progressive Flute", "Flûte (prog.)", "Flauta progresiva"},
      Text{"Progressive Recorder", "Harmonica (prog.)", "Armónica progresiva"}};
   trickNameTable[0xD4] = {
-     Text{"Progressive Titan Blade", "Lame des Titans (prog.)", "Hoja del Titán progresiva"},
-     Text{"Progressive Goron Knife", "Lame Goron (prog.)", "Daga Goron progresiva"},
-     Text{"Progressive Giant Sword", "Épée géante (prog.)", "Espada gigante progresiva"}};
+     Text{"Progressive Darknut Sword", "Épée Darknut (prog.)", "Espada Darknut progresiva"},
+     Text{"Progressive Power Sword", "Épée de puissance (prog.)", "Espada de poder progresiva"},
+     Text{"Progressive Big Stabby", "Gros coup de poignard (prog.)", "Gran puñalada progresiva"}};
 
   trickNameTable[0x0F] = {
-     Text{"Magic Bottle", "Flacon magique", "Frasco feérico"},
-     Text{"Glass Bottle", "Flacon de verre", "Botella de cristal"},
-     Text{"Bottle with Water", "Flacon d'eau", "Botella Tingle"}};
+     Text{"Empty Canteen", "Cantine vide", "cantimplora vacía"},
+     Text{"Vial of Winds", "Fiole de vents", "Vial de Vientos"},
+     Text{"Tingle Bottle", "Flacon de Tingle", "Botella de Tingle"}};
   trickNameTable[0x14] = {
      Text{"Bottle with Chateau Romani", "Flacon de cuvée Romani", "Botella de Reserva Romani"},
-     Text{"Bottle with Fresh Milk", "Flacon de lait frais", "Botella de leche fresca"},
+     Text{"Bottle with Premium Milk", "Flacon avec lait de qualité supérieure", "Biberón con leche Premium"},
      Text{"Bottle with Mystery Milk", "Flacon de lait grand cru", "Botella de leche extra"}};
   trickNameTable[0x8C] = {
      Text{"Bottle with Red Chu Jelly", "Flacon de gelée Chuchu rouge", "Jugo de Chuchu Rojo"},
-     Text{"Bottle with Medicine of Life", "Flacon d'élixir rouge", "Botella de medicina de la vida"},
+     Text{"Bottle with Hibiscus Potion", "Flacon de potion de Hibiscus", "Botella de poción de Hibisco"},
      Text{"Bottle with Heart Potion", "Flacon de potion de soin", "Botella de poción de salud"}};
   trickNameTable[0x8D] = {
      Text{"Bottle with Green Chu Jelly", "Flacon de gelée Chuchu verte", "Jugo de Chuchu Verde"},
-     Text{"Bottle with Medicine of Magic", "Flacon d'élixir vert", "Botella de medicina mágica"},
+     Text{"Bottle with Lamp Oil", "Flacon de Huile à lanterne", "Botella de Aceite de candil "},
```

Comment:
logic consitancy with green, while the oil is yellow :p

Reply:
The temptation to have morshu memes is too strong. :p